parachain-template/pallets/template/src/lib.rs

```diff
@@ -102,5 +115,27 @@ pub mod pallet {
 				},
 			}
 		}
+
+		/// Send an XCM message as parachain sovereign.
+		#[pallet::call_index(2)]
+		#[pallet::weight(Weight::from_parts(100_000_000, 0))]
+		pub fn send_xcm(
```

Comment:
Add a customize xcm call with parachain as origin for smoke tests control related  like `create_channel`.